src/dftbp/solvation/solvparser.F90

```diff
@@ -420,7 +420,7 @@ subroutine readCM5(node, input, geo)
     real(dp) :: conv
     real(dp), allocatable :: atomicRadDefault(:)
 
-    call getChildValue(node, "Alpha", input%alpha, 2.474_dp/AA__Bohr, &
```

Comment:
The usual convention we use is for internal code variables to be in atomic units, and this quantity is an inverse length. The original default value is a conversion is from 1/AA to 1/Bohr (the modifier on line 425 is also converting in this way if there is a user supplied value).

Reply:
As with the `rCutoff`, I simply moved this value. I guess if you think this could introduce a unit error in the future if someone changes the default in cm5.f90, we could change it here already :shrug:

Reply:
We don't want mixed initialization, the input data is defined completely in the parser, this way we have a single source of truth. Adding part of the default values to the derived type would scatter the initialization. Generally, for input data types the parser is place where they are initialized completely, while for container types the `*_init` routine contains all initialization logic.

Reply:
Hmm, I thought the CM5 logic is relatively close to the logic of the Born model, where the defaults are also defined in a separate type and passed to the parser.

If you want to avoid this for the CM5, I can revert the logic in the parser and cm5 and simply throw an error if the API method is called with a missing CM5 input

Reply:
So far DFTB+ does not allow to alter the setup easily from the API, since most logic is embedded in the parser. It would be practical to have the possibility to create and attach containers after the fact to the DFTB+ instance to get additional properties from the API. This requires them to be easy to construct without the parser and self-contained that they needn't be present in the actual calculation (or actually impact the outcome of the calculation).

Reply:
I guess that's what I changed the CM5 to^^

Anyway I don't require to have the case of the missing input covered by DFTB+ for my application since my call already ensures it, so I don't mind either way

Reply:
I don't know how reliable it is to reparse the input and rerun from an API call (might drop some data or perform redundant calculation, but haven't checked). The property you are aiming for is rather cheap compared to a full DFTB calculation.

Also, to get the reparsing and rerunning a lot of code duplication is required, scattering the additional initializations in the code base makes it more difficult to change the CM5 routines in the future.

Reply:
I absolutely agree here (see also my other comment): We are currently not well prepared for a reparsing of the input...